<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbForm</class>
 <widget class="QWidget" name="DbForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>971</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="dbTableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="dbBorderDicomTable">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>991</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="SceneZoom" name="dbGraphicsView">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>0</y>
     <width>441</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SceneZoom</class>
   <extends>QGraphicsView</extends>
   <header location="global">scenezoom.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
